lib/nylas/account.rb

```diff
@@ -33,6 +33,11 @@ def revoke_all(keep_access_token: nil)
       response[:success]
     end
 
+    def ip_addresses
```

Comment:
What are you expecting to be returned from this method?

Reply:
something that looks like this {"updated_at": timestamp, "ip_addresses": [1234, 3457, 2213]}

Reply:
what is the correct variable name for that?

Reply:
Hm okay. Yeah so this is getting into the problem with how this gem was written. I think you're going to have to make more, much less obvious changes including making a new model. We might have to pair on it.